CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Non-GP surrogates not working with `deepcopy` and the simulation module due to slotted
   base class
 
+### Removed
+- Support for Python 3.9
```

Comment:
could we include a link to that recommendation form scientific python?

Reply:
Sure, where would you like to have it? In the changelog with a mention that botorch follows it? Or better place?

Reply:
i think heres nice as users could immediately see why we removed it and wouldnt be anraged by it :D 
somethign like `removed due to new botorch requirements (link1) and recommendations from scientific python (link2)`

Reply:
done. thx